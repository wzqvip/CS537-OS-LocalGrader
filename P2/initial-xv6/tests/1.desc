calling the syscall when no file has been opened.
