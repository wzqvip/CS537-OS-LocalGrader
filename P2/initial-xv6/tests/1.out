XV6_TEST_OUTPUT Open filename: -1
